Xamarin.Forms.Core/ListView.cs

```diff
@@ -210,7 +212,8 @@ public SeparatorVisibility SeparatorVisibility
 			set { SetValue(SeparatorVisibilityProperty, value); }
 		}
 
-		internal ListViewCachingStrategy CachingStrategy { get; private set; }
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Already exposed via interface. Really, we should go back and remove the explicit implementations of the controller interfaces from our code, replace them all with the EditorBrowser idiom, and revert all the callsites back to directly accessing the member.